include/boost/spirit/home/x3/binary.hpp

```diff
@@ -0,0 +1,16 @@
+/*=============================================================================
+    Copyright (c) 2001-2011 Hartmut Kaiser
+
+    Distributed under the Boost Software License, Version 1.0. (See accompanying
+    file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+==============================================================================*/
+#if !defined(BOOST_SPIRIT_X3_BINARY_MAY_08_2007_0906AM)
+#define BOOST_SPIRIT_X3_BINARY_MAY_08_2007_0906AM
+
+#if defined(_MSC_VER)
+#pragma once
```

Comment:
This pragma once for msvc has ben bugging me forever :) 1) is it really that significant? 2) are there no pragma once in other compilers. this is quite prevalent in spirit.